packages/terra-docs/.npmignore

```diff
@@ -0,0 +1,8 @@
+*.log
```

Comment:
Should we start adding a "files" key to the package.json to only release docs, lib, and tests?

Reply:
we need to release src too for code examples

Reply:
right. Still might be less to configure than adding all generated files & config files to npmignore

Reply:
The files key can be a little dangerous too, there's been a few situations where a new file is created and is ignored during the publish that crashes the package. These situations are rare, but I don't know of a good way to detect them automatically.